### PR TITLE
amu_roboclub_team.json

### PR DESCRIPTION
**Pull request by irfankhaan**

### CODE REVIEW

**Comment by irfankhaan** [comment]
@iamareebjamal please review my PR

**Comment by iamareebjamal** [comment]
Will do

File: amu_roboclub_team.json [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    		"session" : "2019-20"
+    	},
+    	"members" : {
+    		"abidalikhan" : {
+    			"name" : "Dr. Abid Ali Khan",
+    			"position" : "Faculty Advisor",
+    			"rank" : 20,
+    			"thumbnail" : "https://res.cloudinary.com/amuroboclub/image/upload/old/team/DrAbidAliKhan.bmp"
+    		},
+    		"anwawsadat" : {
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]: Wrong

File: amu_roboclub_team.json [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    		"ekramkhan" : {
+    			"links" : {
+    				"email" : "ekhan.el@amu.ac.in"
+    			},
+    			"name" : "Prof. Ekram Khan",
+    			"position" : "Teacher Incharge",
+    			"rank" : 10,
+    			"thumbnail" : "https://res.cloudinary.com/amuroboclub/image/upload/old/team/ProfEkramKhan.jpg"
+    		},
+    		"mumtazahmadkhan" : {
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]: Change accordingly

File: amu_roboclub_team.json [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    			"uid" : "VNSHqA56CIhqgkuo9YbHYxk5z7n1"
+    		},
+    		"kakulsrivastava" : {
+    			"links" : {
+    				"email" : "kakulshrivastava.1@gmail.com",
+    				"facebook" : "https://www.facebook.com/kakul.shrivastava.1",
+    				"linkedin" : "https://www.linkedin.com/in/kakul-shrivastava-821310148/",
+    				"mobile" : "9634747915"
+    			},
+    			"name" : "Kakul Srivastava",
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]:
```suggestion
    			"name" : "Kakul Shrivastava",
```

File: amu_roboclub_team.json [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+    			"old_avatars" : {
+    				"-KmFC5EJb4hUe4G7vPav" : "https://www.materialui.co/materialIcons/social/person_grey_192x192.png",
+    				"-Ks4yELT5Xom3shuFARK" : "https://res.cloudinary.com/amuroboclub/image/upload/v1497067967/profile_img/12042681_900308980065236_9069563880213364885_n.jpg",
+    				"-LVP0B43CxnH48dQGgT1" : "https://res.cloudinary.com/amuroboclub/image/upload/v1503338730/profile_img/AirBrush_20170821233516.jpg"
+    			},
+    			"position" : "Coordinator",
+    			"profile_info" : {
+    				"batch" : "B.Tech (Electronics Engineering)\n2016-20",
+    				"cv" : "https://drive.google.com/file/d/1--_NoGR0fWKosxmhjQgzL3HuPyB6NOsY/view?usp=drivesdk",
+    				"interests" : ["Embedded systems", "Robotics"]
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]:
```suggestion
    				"interests" : ["Embedded Systems", "Robotics"]
```

File: amu_roboclub_team.json [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+    				"mobile" : "7500957679"
+    			},
+    			"name" : "Mayank Gupta",
+    			"old_avatars" : {
+    				"-LFhy3CjBvdmlWl7QdlL" : "https://www.materialui.co/materialIcons/social/person_grey_192x192.png"
+    			},
+    			"position" : "Joint Coordinator",
+    			"rank" : 40,
+    			"thumbnail" : "https://res.cloudinary.com/amuroboclub/image/upload/v1554290001/profile_img/IMG_20180630_125455_789.jpg",
+    			"uid" : "IHJgzo9GjmVBCSTIpHjzLsSwDAJ2"
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]: Why is this UID changed?

File: amu_roboclub_team.json [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+    				"batch" : "B.Tech (Computer sciences) \n2017-2021",
+    				"interests" : [ "Computer programming ", "Mechanical fabrication" ]
+    			},
+    			"rank" : 40,
+    			"thumbnail" : "https://res.cloudinary.com/amuroboclub/image/upload/v1529774763/profile_img/PicsArt_10-19-01.24.12.jpg",
+    			"uid" : "pqeRvjRGbnO4qYk54ScIoNIyNP13"
+    		},
+    		"sheikhsaud" : {
+    			"links" : {
+    				"emails" : "saudredmi@gmail.com",
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]:
```suggestion
    				"email" : "saudredmi@gmail.com",
```

**Comment by irfankhaan** [comment]
Sorry! I'll correct these mistakes.

File: amu_roboclub_team.json (hunk 2 above)
Comment (irfankhaan) [reply amu_roboclub_team.json]: What changes do I have to make in this?

File: amu_roboclub_team.json [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+    		},
+    		"priyankagupta" : {
+    			"name" : "Priyanka Gupta",
+    			"position" : "Volunteer",
+    			"rank" : 70,
+    			"thumbnail" : "https://www.materialui.co/materialIcons/action/account_circle_grey_192x192.png"
+    		},
+    		"shilpyvarshney" : {
+    			"links" : {
+    				"emails" : "shilpyvarshney2000@gmail.com"
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]: Still wrong

File: amu_roboclub_team.json [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+    			},
+    			"name" : "Sonal Gupta",
+    			"position" : "Volunteer",
+    			"rank" : 70,
+    			"thumbnails" : "https://www.materialui.co/materialIcons/action/account_circle_grey_192x192.png",
+    			"uid" : "KPhxsOufO7VB86xatXZUe5hxoq42"
+    		},
+    		"vivekrajsingh" : {
+    			"links" : {
+    				"emails" : "vivekrajs3999@gmail.com",
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]: Still wrong

File: amu_roboclub_team.json (hunk 2 above)
Comment (iamareebjamal) [reply amu_roboclub_team.json]: badrulhasankhan

**Comment by irfankhaan** [comment]
@iamareebjamal, sorry if this caused any delay!

File: amu_roboclub_team.json [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+    	"members" : {
+    		"abidalikhan" : {
+    			"name" : "Dr. Abid Ali Khan",
+    			"position" : "Faculty Adviser",
+    			"rank" : 20,
+    			"thumbnail" : "https://res.cloudinary.com/amuroboclub/image/upload/old/team/DrAbidAliKhan.bmp"
+    		},
+    		"anwarsadat" : {
+    			"links" : {
+    				"email" : "anwar7039@gmail.com"
```
Comment (iamareebjamal) [inline amu_roboclub_team.json]: Why is there a problem in copy/pasting?

You changed it from correct email to wrong one

Comment (irfankhaan) [reply amu_roboclub_team.json]: I didn't copy paste and it's the same email in the file which Haider sent me. Under the 2018-19 session.

Comment (irfankhaan) [reply amu_roboclub_team.json]: I didn't copy paste.

Comment (iamareebjamal) [reply amu_roboclub_team.json]: That was the problem. Why didn't you copy/paste. These are not the things you write by hand

**Pull request closed without merging**